docs/product/base/grid.html

```diff
@@ -0,0 +1,816 @@
+        <table class="wmn4 s-table s-table__bx-simple">
+            <thead>
+                <tr>
+                    <th class="s-table--cell3" scope="col">Class</th>
+                    <th class="s-table--cell3" scope="col">Output</th>
+                    <th scope="col">Definition</th>
+                    <th class="s-table--cell2 ta-center" scope="col"><a class="s-link s-link__inherit" href="{{ "/product/guidelines/conditional-classes#responsive" | relative_url }}">Responsive?</a></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for item in grid.alignment %}
+                    <tr class="fs-caption">
+                        <th scope="row"><code class="stacks-code">.{{ item.class }}</code></th>
+                        <td><code class="stacks-code bg-white">{{ item.output }}</code></td>
+                        <td>{{ item.define }}</td>
+                        <td class="ta-center">
+                            {% if item.responsive %}
+                                {% icon "Checkmark", "fc-green-500" %}
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+
+    {% header "h3", "Alignment examples" %}
+    <div class="stacks-preview">
+{% highlight html %}
+<div class="d-grid ji-center ai-center">
+    <div class="grid--item as-start">…</div>
+    <div class="grid--item">…</div>
+    <div class="grid--item js-end">…</div>
+    …
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example ff-mono">
+            <div class="mb16">Default</div>
+            <div class="d-grid g-af-column mb24">
+                <div class="grid--item bg-black-075 p12 ba bc-black-3">1</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3">2</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3">3</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3">4</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3">5</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3">6</div>
+            </div>
+
+            <div class="mb16">.ji-center .ai-center</div>
+            <div class="d-grid g-af-column ji-center ai-center h96">
+                <div class="grid--item bg-black-075 p12 ba bc-black-3 h48 as-start">1 .as-start</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3 h64">2</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3 as-stretch">3 .as-stretch</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3 h64 js-end">4 .js-end</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3 h64">5</div>
+                <div class="grid--item bg-black-075 p12 ba bc-black-3 h64">6</div>
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
```suggestion
</section>
<style>
    @media (max-width: 640px) {
        .sm\:highlight {
            padding: 3px 8px;
            background-color: var(--highlight-variable);
            border-radius: 40px;
            color: var(--highlight-bg);
        }
    }
</style>
```

Reply:
`.sm:highlight` wasn't doing anything without this. I C&P'd this from elsewhere, but it might be worth considering moving this up to `stacks-documentation.less`.